actors/builtin/init/init_test.go

```diff
@@ -76,7 +76,7 @@ func TestExec(t *testing.T) {
 
 		var st init_.State
 		rt.GetState(&st)
-		actualIdAddr, err := st.ResolveAddress(rt.Store(), uniqueAddr1)
+		actualIdAddr, err := st.ResolveAddress(adt.AsStore(rt), uniqueAddr1)
```

Comment:
De we actually need to keep `adt.AsStore`?

Reply:
This is annoying. The mock was already constructed not to require this explicit wrapping. Please restore a method to the mock runtime that can be reached directly by test code.

@magik6k if we do https://github.com/filecoin-project/specs-actors/issues/140 then we won't need any wrapping anywhere.